pkg/apis/crd/v1alpha1/types.go

```diff
@@ -868,3 +874,21 @@ type BundleServerAuthConfiguration struct {
 	// AuthSecret is a Secret reference which stores the authentication value.
 	AuthSecret *v1.SecretReference `json:"authSecret"`
 }
+
+type L7Protocol struct {
+	HTTP *HTTPProtocol `json:"http,omitempty"`
+}
+
+// HTTPProtocol matches HTTP requests with specific host, method, and path. All
+// fields could be used alone or together. If all fields are not provided, this
+// matches all HTTP requests.
+type HTTPProtocol struct {
+	// Host represents the hostname present in the URI or the HTTP Host header to match.
```

Comment:
Should we use "URL" or "URI"?

Reply:
Used URI following Suricata's doc: https://suricata.readthedocs.io/en/suricata-6.0.0/rules/http-keywords.html#id2

> The http.host and http.host.raw buffers are populated from either the URI (if the full URI is present in the request like in a proxy request) or the HTTP Host header. If both are present, the URI is used.

Reply:
Ok.